daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala

```diff
@@ -415,9 +413,7 @@ class TransactionCoderSpec
       }
     }
 
-    // FIXME: https://github.com/digital-asset/daml/issues/7709
-    // replace all occurrences of "dev" by "11", once "11" is released
-    "ignore field observers in version < dev" in {
+    "ignore field observers in version < 11" in {
```

Comment:
```suggestion
    "ignore observers field in version < 11" in {
```
"field observers" sounds like a noun rather than "the field named 'observers'".